docs/development/contributing.rst

```diff
@@ -75,35 +97,42 @@ If you would like to contribute on a regular basis, you'll want to read the
 development environment to preview changes, run tests locally, etc.
 
 
+
 Technical Writers
 ~~~~~~~~~~~~~~~~~
 
 Technical writers and editors are invited to review the `documentation
 <https://docs.securedrop.org/>`__ and fix any mistakes in accordance with the
-:doc:`documentation guidelines <documentation_guidelines>`.
+:doc:`documentation guidelines <documentation_guidelines>`. Our documentation code is 
+located in our `documentation repository <https://github.com/freedomofpress/securedrop-docs>`__.
 
-If this is your first time helping with SecureDrop documentation, consider
-working on `low-hanging fruit`_ to become familiar with the process.
+If this is your first time contributing to SecureDrop documentation, consider
+working on `low-hanging fruit`_ to become familiar with the process. Refer to our
```

Comment:
```suggestion
working on a `low-hanging fruit`_ to become familiar with the process. Refer to our
```

Again, not 100% sure if this suggestion should be applied, but it seems to me that "a _low-hanging fruit_" (or "_low-hanging fruits_") fit better in the sentence?

Reply:
@gonzalo-bulnes "Fruit" is [usually uncountable](https://www.ldoceonline.com/dictionary/fruit), so in English the sentence as written works better than your suggested change. This PR currently breaks that link by removing the link target, however.